pkg/api/testing/fuzzer.go

```diff
@@ -301,6 +302,9 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 			c.FuzzNoCustom(n)
 			n.Spec.ExternalID = "external"
 		},
+		func(s *expapi.APIVersion, c fuzz.Continue) {
+			s.APIGroup = "something"
```

Comment:
We need to use "something". c.RandString() sometimes generate empty string, which causes the test error.